content/tutorials/_includes/upload_manual.md

```diff
@@ -1,8 +1,10 @@
-Let's get back to FrontLine. To navigate to the artifacts page, click on the {{< icon archive >}} icon on the Sidenav (navigation bar on the left of the screen).
+Let's get back to Gatling Enterprise. 
+To navigate to the artifacts page, click on the **Artifacts** on the left navigation bar.
 
 {{< img src="artifact_navigation.png" alt="Sidenav navigation" >}}
 
-Click on the create button in order to create an artifact in FrontLine. Choose the name you want, then click save.
+Click on the create button in order to create an artifact. 
+Choose the name you want, then click save.
 
 {{< img src="create_artifact_button.png" alt="Create an artifact" >}}
 
```

Comment:
Line 11: "You now have to upload the artifact you generated earlier. Click on the {{< icon cloud-upload-alt >}} icon."